### ISSUE TITLE
no method matching readline(::SerialPort, ::Float64)

### ISSUE THREAD

**Issue by AndiMD** [JuliaIO/LibSerialPort.jl#46]
Hi there,

the first example with timeouts is broken in Julia 1.3, Linux 64bit.
```
using LibSerialPort
LibSerialPort.open("/dev/ttyACM0", 115200) do s
    ret = readline(s, 1.0) #try to readline with a 1 second timeout
    @show ret
end
```
```ret = readline(s, 1.0) #try to readline with a 1 second timeout
ERROR: LoadError: MethodError: no method matching readline(::SerialPort, ::Float64)
Closest candidates are:
  readline(::IO; keep) at io.jl:454
```

`readuntil(s, '\n', 1.0)` still works, so its not a big deal.

Cheers,
Andi

**Comment by andrewadare** [comment]
This is due to some activity around #45 that we need to smooth out. Sorry for the trouble, we will get it straightened out soon. It sounds like you have found a workaround that is getting you along in the meantime.

**Comment by ianshmean** [comment]
Now available in 0.4
